protos/camera/camera.proto

```diff
@@ -275,22 +346,30 @@ message CameraStatus {
     StorageStatus storage_status = 8; // Storage status
 }
 
+// Type to represent a setting option.
+message Option {
+    string option_id = 1; // Name of the option (machine readable)
+    string option_description = 2; // Description of the option (human readable)
+}
+
 // Type to represent a setting with a selected option.
 message Setting {
     string setting_id = 1; // Name of a setting (machine readable)
     string setting_description = 2; // Description of the setting (human readable)
     Option option = 3; // Selected option
-}
-
-// Type to represent a setting option.
-message Option {
-    string option_id = 1; // Name of the option (machine readable)
-    string option_description = 2; // Description of the option (human readable)
+    bool is_range = 4; // If option is given as a range
```

Comment:
So now this is like `setting_description`, right? I.e. when the user creates the object, they should ignore this field. Correct?

Reply:
What about I add a default value for them (`""` and `false`)? Good enough?